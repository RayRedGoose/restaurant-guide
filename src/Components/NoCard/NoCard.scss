@import "../../assets/css/variables.scss";

.no-card {
  background-color: #fff;
  border-radius: 1rem;
  box-shadow: 0px 0px 12px rgba(23, 126, 137, 0.2);
  display: flex;
  align-items: center;
  justify-content: center;
  padding: 3rem;
  width: 80%;
  margin: auto;
  img {
    width: 5rem;
    margin-right: 3rem;
  }
}
