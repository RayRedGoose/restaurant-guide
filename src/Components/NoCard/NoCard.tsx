import './NoCard.scss';
import React from 'react';
import cafe from 'assets/images/cafe.svg';

const NoCard: React.FC = () => {
  return (
    <section className="no-card">
      <img src={cafe} alt="cafe"/>
      <h2>No restaurant to display!</h2>
    </section>
  );
};

export default NoCard;
