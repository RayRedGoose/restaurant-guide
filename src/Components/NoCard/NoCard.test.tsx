import React from 'react';
import { shallow } from 'enzyme';
import NoCard from './NoCard';

it('should match the snapshot', () => {
  const wrapper = shallow(<NoCard />);
  expect(wrapper).toMatchSnapshot();
});
