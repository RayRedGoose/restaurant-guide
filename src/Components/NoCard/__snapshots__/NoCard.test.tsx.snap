// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<section
  className="no-card"
>
  <img
    alt="cafe"
    src="cafe.svg"
  />
  <h2>
    No restaurant to display!
  </h2>
</section>
`;
